### ISSUE TITLE
The "Firefox Notes" sidebar has a noticeable delay when it's reopened or switched

### ISSUE THREAD

**Issue by Softvision-MariusComan** [mozilla/notes#401]
**[Affected versions]:**
- Firefox 57 and up

**[Affected Platforms]:**
- All Windows
- All Mac
- All Linux

**[Prerequisites]:**
- Have a Firefox profile with the latest "Firefox Notes" add-on version (1.9.0b1 build with CKeditor) installed.
- Have the "Firefox Notes" sidebar closed.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites.
2. Click the "Show sidebars" button from the browser toolbar and observe the sidebar behavior.

**[Expected result]:**
- The sidebar is displayed almost instantly.

**[Actual result]:**
- The sidebar is displayed after a delay.

**[Notes]:**
- The issue is not reproducible with the "Firefox Notes" master build.
- The issue is reproducible only with the "Firefox Notes" sidebar.
- Attached a screen recording of the issue:
![rec of the issue](https://user-images.githubusercontent.com/31243259/33022047-736d9ab6-ce0c-11e7-8448-fbed48862f62.gif)

**Comment by vladikoff** [comment]
Should be faster with multi-note and latest CKEditor

**Comment by Softvision-MariusComan** [comment]
I have retested this issue using the latest version of the "Firefox Notes" add-on (4.2.0dev custom built on 21.06.2018) installed, on the latest Firefox release and Nightly and it seems that the original behavior has not been improved. 

I've made a screen recording using the "ScreenToGif" software, at 60fps and 1302x1009px in order to find out how big the delay in frames is. Here are the results:

with 1 note in the sidebar - it is reopened in 5 frames
with 3 notes in the sidebar - it is reopened in 6 frames
with 10 notes in the sidebar - it is reopened in 8 frames

I mention that with the build (1.9.0b1) from the original issue the sidebar is reopened in 6 frames. 
Also, both tests were performed without being logged into "Firefox Notes"

Considering this I am reopening this issue.
